playdate/sound.lua

```diff
@@ -1,15 +1,50 @@
 -- docs: https://sdk.play.date/2.6.2/Inside%20Playdate.html#M-sound
 
-playdate.sound = {}
+local module = {}
+playdate.sound = module
+
+function module.getSampleRate()
+  return 44100.0 -- hardcoded by Playdate hardware
+end
+
+function module.playingSources()
+  error("[ERR] playdate.sound.playingSources() is not yet implemented.")
+end
+
+function module.addEffect(effect)
+  error("[ERR] playdate.sound.addEffect() is not yet implemented.")
+end
+
+function module.removeEffect(effect)
+  error("[ERR] playdate.sound.removeEffect() is not yet implemented.")
+end
+
+function module.getHeadphoneState(changeCallback)
+  error("[ERR] playdate.sound.getHeadphoneState() is not yet implemented.")
+end
+
+function module.setOutputsActive(headphones, speaker)
+  error("[ERR] playdate.sound.setOutputsActive() is not yet implemented.")
+end
+
+function module.getCurrentTime()
+  error("[ERR] playdate.sound.getCurrentTime() is not yet implemented.")
+end
+
+function module.resetTime()
+  error("[ERR] playdate.sound.resetTime() is not yet implemented.")
+end
 
 local sampleplayer = {}
 playdate.sound.sampleplayer = sampleplayer
 sampleplayer.meta = {}
 sampleplayer.meta.__index = sampleplayer.meta
 
+-- TODO: handle overloaded signature (sample) - sample is a playdate.sound.sample
 function sampleplayer.new(path)
   local sample = setmetatable({}, sampleplayer.meta)
   sample.data = love.audio.newSource(path..".wav", "static")
+  -- TODO: should we force a sample rate of 44100.0 since Playdate's hardware does this?
   return sample
```

Comment:
@fletchmakes Ohh good catch! I don't think we can do this at runtime as I can't find a relevant API (there is a [get sample rate API](https://love2d.org/wiki/SoundData:getSampleRate)) but we can enforce this at build time by forcing ffmpeg to output files at 44100 by default. If users want to override this, they can just change the file processor.

I created #35 to capture this - let's link the issue in the comment for posterity

```suggestion
  -- TODO: force audio to 44100 at build time https://github.com/GamesRightMeow/playbit/issues/35
  return sample
```